addons/stock/models/stock_move_line.py

```diff
@@ -921,9 +921,7 @@ def action_open_reference(self):
         }
 
     def action_put_in_pack(self):
-        for picking in self.picking_id:
-            picking.with_context(move_lines_to_pack_ids=self.ids).action_put_in_pack()
-        return True
+        return self.picking_id.with_context(move_lines_to_pack_ids=self.ids).action_put_in_pack()
```

Comment:
Maybe this would be more user friendly?

```suggestion
        if len(self.picking_id) > 1:
           raise UserError(_("You cannot directly pack quantities from different transfers into the same package through this view. Try adding them to a batch picking and pack it there.")
        return self.picking_id.with_context(move_lines_to_pack_ids=self.ids).action_put_in_pack()
```

Reply:
Done! Thank you for your review ;)